src/components/MoveSet/MoveSet.js

```diff
@@ -1,27 +1,33 @@
 import './MoveSet.scss'
 import { registerHtml } from "tram-one"
 import MoveDropdown from '../MoveDropdown'
-import usePokemon from '../../hooks/usePokemon'
 
 const html = registerHtml({
   MoveDropdown
 })
 
-export default ({index}) => {
-  const { pokemon } = usePokemon()
+export default ({ pokemon, pokemonMoveset, pokemonMoves, onUpdatePokemonMove }) => {
   if (!pokemon) {
     return ''
   }
 
+  const moveOptions = pokemon && pokemon.moves.map(({move}) => move.name)
+
+  const selectedNames = [pokemonMoveset[0], pokemonMoveset[1], pokemonMoveset[2], pokemonMoveset[3]]
```

Comment:
this array seems weird

Reply:
I'm not a fan either, but for some reason, the pokemonMoveset is an object